### DISCUSSION TITLE
Problems setting PIN with BluetoothSerial lib.

### DISCUSSION THREAD

**Discussion by Alain-Godo** [espressif/arduino-esp32#8342] [Q&A]
Hi, I'm using the BluetoothSerial lib, I provided a pin for the connection but the devices don't ask for it in the pairing process, it's like the pin is not really set with the .setPin() method. What's wrong?

**Comment by Alain-Godo** [comment]
Does nobody know what's happened? Because the slave example with PIN of the documentation it's not working:
https://espressif-docs.readthedocs-hosted.com/projects/arduino-esp32/en/latest/api/bluetooth.html

**Comment by VojtechBartoska** [comment]
@PilnyTomas Can you please take a look on this? Maybe it's caused by some example updates.
